MAINTAINERS.md

```diff
@@ -0,0 +1,34 @@
+# Maintainers
+
+This page includes information for maintainers of this repo.
+
+## How to manually generate the binaries for a Github release
+
+The binaries for the Github release are generated using [GoReleaser](https://goreleaser.com/). The Github workflow [ci-release.yml](./.github/workflows/ci-release.yml) should automatically create pre-build binaries and attach them to the release. However, if the Github workflow fails, you can manually generate pre-built binaries and attach them to the release.
```

Comment:
```suggestion
The binaries for the Github release are generated using [GoReleaser](https://goreleaser.com/). The Github workflow [ci-release.yml](./.github/workflows/ci-release.yml) should automatically create pre-build binaries and attach them to the release.
```

If the pipeline fails, it should be fixed. Either there is a bug in the action or there are tests that are failing. 
Documenting that it is ok to just ignore the failure feels like bad practice.

Reply:
> Documenting that it is ok to just ignore the failure feels like bad practice.

This PR does not intend to document that it is ok to ignore pipeline failures. I moved these steps from the README.md to MAINTAINERS.md b/c these steps should ideally never be used. They are only necessary if the pipeline does fail. I don't think we need the manual steps if we expect the pipeline to always work.

Reply:
Does your suggestion include deleting the steps below?